src/test/java/uk/gov/companieshouse/api/strikeoffobjections/model/chipsrestinterfaceskafkamessage/ChipsRestInterfacesKafkaMessageContentTest.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.chipsrestinterfaceskafkamessage;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.Test;
+
+import uk.gov.companieshouse.api.strikeoffobjections.groups.Unit;
+import uk.gov.companieshouse.api.strikeoffobjections.utils.Utils;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Unit
+public class ChipsRestInterfacesKafkaMessageContentTest {
+
+    private static final String APP_ID = "APP_ID";
+    private static final String MESSAGE_ID = "MESSAGE_ID";
+    private static final String CHIPS_REST_ENDPOINT = "ENDPOINT";
+    private static final LocalDateTime CREATED_AT =
+            LocalDateTime.of(2020, 1, 1, 0, 0);
+
+    @Test
+    void toStringTest() throws JsonProcessingException {
```

Comment:
I don't see why JsonProcessingException would need to be in the signature here.
What in the method could throw this exception?